securedrop/sass/_base.sass

```diff
@@ -28,6 +28,9 @@
     text-align: left
     font-size: 30px
 
+    &:dir(rtl)
+      text-align: right
+
```

Comment:
A fine approach. We can get fancy with SASS variables on a rainy day :+1:

Reply:
As I mentioned [here](https://github.com/freedomofpress/securedrop/issues/2381#issuecomment-333780521), I think that won't work quite as slick as we need because the variables are static after they're compiled, so we'd still have to use jinja at a minimum to pull down the right CSS. I feel like there has to be a better way to do this, but I could be wrong.